src/_cffi_src/openssl/ssl.py

```diff
@@ -688,6 +689,12 @@
 
 static const long Cryptography_HAS_SSL_CTX_CLEAR_OPTIONS = 1;
 
+#ifdef OPENSSL_NO_TLSEXT
```

Comment:
Correct. By using cryptography as the basis of the module you're bound by the versions we've chosen to support, but OPENSSL_NO_TLSEXT is extremely uncommon and in practice we've never had a single bug report complaining about not supporting that configuration.